assets/sass/globals/_base.scss

```diff
@@ -2,7 +2,7 @@
    Base & Element level Styles
    ========================================================================= */
 
-[v-cloak] {
+.js-on [v-cloak] {
```

Comment:
Is this right? We absolutely want to hide things with `v-cloak` for non-JS users too, right?